litmus-portal/k8s-manifest.yml

```diff
@@ -1,9 +1,3 @@
----
-apiVersion: v1
```

Comment:
Why is this removed? The litmus ns needs to be created separately then.

Reply:
I have added it back, we will have a custom script for okteto. to make changes in our portal yaml.